core/commands/add.go

```diff
@@ -29,6 +31,7 @@ const progressReaderIncrement = 1024 * 256
 const (
 	progressOptionName = "progress"
 	wrapOptionName     = "wrap-with-directory"
+	hiddenOptionName   = "hidden"
```

Comment:
On Thu, May 07, 2015 at 01:27:53PM -0700, Gaëtan Voyer-Perrault wrote:
> > I'm not sure how they handle Windows' hidden attribute
> 
> Just tested this locally and `git` respects the hidden attribute. It
> also creates the `.git` folder as a hidden folder.

That's probably a good model to emulate then.  With IPFS, it would be
possible to store information like this in a Metadata wrapper around
files and directories, but I'm not sure how much like Window's native
filesystem we need IPFS to be.

Does Git provide a way to explicitly add hidden files and directories?
If they don't, I'd suggest we follow their lead in that too, at least
until we get around to storing that attribute in a Metadata wrapper.